t/lang.t

```diff
@@ -79,4 +79,22 @@ sub test_logo_exists {
 test_logo_exists('logo');
 test_logo_exists('logo2x');
 
+# Test that {variables} are kept in translations
+
+foreach my $stringid (sort keys %Lang) {
+	while ($Lang{$stringid}{en} =~ /{([^}]+)\}/g) {
+		my $variable = $1;
+		foreach my $l (sort keys %{$Lang{$stringid}}) {
```

Comment:
using $l is can confuse hear (I read $1 at first thight…).
maybe replace with $language ?